api/proto/node/node.proto

```diff
@@ -108,16 +108,19 @@ message Bundle {
 message BlobHeader {
 	// The KZG commitment to the polynomial representing the blob.
 	bytes commitment = 1;
+  	// The KZG commitment to the polynomial representing the blob on G2, it is uesd 
```

Comment:
`uesd` -> `used`.
The length_proof is used to prove the degree, not length_commitment?